### PR TITLE
change title on README

### PR DESCRIPTION
**Pull request by dinwa-ms**
Left reviews and line comments
Created a file and made a commit
Opened a pull request

### CODE REVIEW

**Review by dinwa-ms — COMMENTED** [review]
Change title

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
```
Comment (torrac) [inline README.md]: some update

**Review by dinwa-ms — COMMENTED** [review]
changefd

File: README.md (hunk 1 above)
Comment (torrac) [inline README.md]: Left reviews and line comments
Created a file and made a commit
Opened a pull request

**Comment by torrac** [comment]
Left reviews and line comments
Created a file and made a commit
Opened a pull request

**Pull request merged**